<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="ja-jp">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">07f6d4a4553a3657a7f893c23d9b8c6d2333e6da</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.ApplicationException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">bafec8cb6f9e5b25c0c60665c967f121ede0dd42</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f193e60bcfe98944a507c7a529b47961a30cf755</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>Serves as the base class for application-defined exceptions.</source>
          <target state="translated">アプリケーション定義の例外の基底クラスとして機能します。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>クラスです。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error, such as "An application error has occurred."</source>
          <target state="translated">このコンス トラクターは、 &lt;xref:System.Exception.Message%2A&gt;「アプリケーション エラーが発生しました」などに、エラーを説明するシステム提供のメッセージの新しいインスタンスのプロパティ&lt;/xref:System.Exception.Message%2A&gt;を初期化します</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">このメッセージでは、現在のシステム カルチャが考慮されます。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ApplicationException&gt;.</source>
          <target state="translated">次の表は、 &lt;xref:System.ApplicationException&gt;。&lt;/xref:System.ApplicationException&gt;のインスタンスの初期プロパティ値を示しています。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|</source>
          <target state="translated">|プロパティ |値 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|Null 参照 (<ph id="ph1">`Nothing`</ph> Visual Basic で). |  |&lt;xref:System.Exception.Message%2A&gt;|ローカライズされたエラー メッセージ文字列 |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt; 。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>指定したエラー メッセージを使用します。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated">内容、<ph id="ph1">`message`</ph>パラメーターをユーザーが理解できるものではします。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">このコンス トラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認するために必要です。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">このメッセージでは、現在のシステム カルチャが考慮されます。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ApplicationException&gt;.</source>
          <target state="translated">次の表は、 &lt;xref:System.ApplicationException&gt;。&lt;/xref:System.ApplicationException&gt;のインスタンスの初期プロパティ値を示しています。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |<ph id="ph2">`Message`</ph>|The error message string.|</source>
          <target state="translated">|プロパティ |値 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|Null 参照 (<ph id="ph1">`Nothing`</ph> Visual Basic で). |  |<ph id="ph2">`Message`</ph>|エラー メッセージ文字列 |&lt;/xref:System.Exception.InnerException%2A&gt; 。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>A message that describes the error.</source>
          <target state="translated">エラーを説明するメッセージ。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>シリアル化されたデータを持つクラス。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">このコンス トラクターは、ストリームで送信された例外オブジェクトを再構成する逆シリアル化中に呼び出されます。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</source>
          <target state="translated">詳細については、次を参照してください。 <bpt id="p1">[</bpt>XML および SOAP シリアル化<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>です。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">シリアル化されたオブジェクト データを保持するオブジェクト。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">ソースまたは転送先に関する文脈情報。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">新しいインスタンスを初期化、 <bpt id="p1">&lt;xref href="System.ApplicationException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>指定したエラー メッセージおよびこの例外の原因となった内部例外への参照を持つクラス。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated">内容、<ph id="ph1">`message`</ph>パラメーターをユーザーが理解できるものではします。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">このコンス トラクターの呼び出し元は、この文字列が現在のシステムのカルチャに合わせてローカライズ済みであることを確認するために必要です。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">前の例外への参照を含める必要があります前の例外の直接の結果としてスローされる例外、&lt;xref:System.Exception.InnerException%2A&gt;プロパティ&lt;/xref:System.Exception.InnerException%2A&gt;。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or a null reference if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">&lt;xref:System.Exception.InnerException%2A&gt;プロパティが同じ場合は、コンス トラクター、または null 参照に渡される値を返します、&lt;xref:System.Exception.InnerException%2A&gt;プロパティがコンス トラクターには、内部例外値を指定していません&lt;/xref:System.Exception.InnerException%2A&gt;&lt;/xref:System.Exception.InnerException%2A&gt;。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ApplicationException&gt;.</source>
          <target state="translated">次の表は、 &lt;xref:System.ApplicationException&gt;。&lt;/xref:System.ApplicationException&gt;のインスタンスの初期プロパティ値を示しています。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">|プロパティ |値 |  |--------------|-----------|  |&lt;xref:System.Exception.InnerException%2A&gt;|内部例外の参照 |。  |&lt;xref:System.Exception.Message%2A&gt;|エラー メッセージ文字列 |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt; 。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">例外の原因を説明するエラー メッセージ。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">現在の例外の原因となった例外。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not a null reference, the current exception is raised in a <bpt id="p1">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">場合、<ph id="ph1">`innerException`</ph>パラメーターが null 参照でないで現在の例外が発生、 <bpt id="p1">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>ブロックは、内部例外を処理します。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>